netlify-plugin-cloudinary/src/data/errors.ts

```diff
@@ -6,5 +7,8 @@ export const ERROR_INVALID_IMAGES_PATH = 'Invalid asset path. Please make sure y
 export const ERROR_NETLIFY_HOST_CLI_SUPPORT =
     'Note: The Netlify CLI does not currently support the ability to determine the host locally, try deploying on Netlify.';
 export const ERROR_NETLIFY_HOST_UNKNOWN =
+    'Cannot determine Netlify host, can not proceed with plugin.'
+export const ERROR_SITE_NAME_REQUIRED =  'Cannot determine the site name, can not proceed with plugin'
+export const ERROR_UPLOAD_PRESET = 'To use a delivery type of "upload", please use an uploadPreset for unsigned requests or an API Key and Secret for signed requests'
+export const ERROR_INVALID_SRCSET = 'Invalid srcset path. Please make sure the srcset is defined.'
     'Cannot determine Netlify host, can not proceed with plugin.';
```

Comment:
hey i think something funky is going on here in this file - it looks like there's an extra line that's just a string?